.github/workflows/main.yml

```diff
@@ -113,22 +118,83 @@ jobs:
         run: |
           python3 -m cibuildwheel --output-dir ${{matrix.builddir1}}
 
-      - name: Fix windows wheels
-        if: contains(matrix.os,'windows')
+      - name: Upload wheels as artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.os }}-wheels
+          path: ${{matrix.builddir1}}/
+          retention-days: 5
+
+  windows-tests:
+    name: windows py${{ matrix.python-version }} tests
+    runs-on: windows-latest
+    needs: build-wheels
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+        include:
+          - os: windows-latest
+            triplet: x64-windows
+            openmp: ON
```

Comment:
This line can be deleted right? Thats just a variable I added in that was used for the build system, doesn't actually do any magic.

Reply:
fixed in c56ce2b1b2b1be4b959af76334d4fd41a3a00f0a